src/plays/savings-calculator/components/SavingOptions/index.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import { getCalculatedSaving } from "../../utils/getCalculatedSaving";
+import "./savingOptions.scss";
+
+function SavingOptions(props) {
+  const { currency, setCurrency, setTotal } = props;
+
+  const [startingBalance, setStartingBalance] = useState(1000);
+  const [monthlyContribution, setMonthlyContribution] = useState(100);
+  const [period, setPeriod] = useState(1);
+  const [periodChoice, setPeriodChoice] = useState("Years");
+  const [annualInterest, setAnnualInterest] = useState(8);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const total = getCalculatedSaving({
+      startingBalance,
+      monthlyContribution,
+      period,
+      periodChoice,
+      annualInterest,
+    });
+
+    setTotal(total);
+  };
+
+  return (
+    <div className="savingOptions">
+      <div className="savingOptions__currencyGroup">
+        <label htmlFor="currency">Currency:</label>
+        <select
+          className="savingOptions__currencies"
+          id="currency"
+          value={currency}
+          onChange={(e) => setCurrency(e.target.value)}
+        >
+          <option>INR</option>
+          <option>USD</option>
```

Comment:
@kd100100, This will not work as we have not added the value attribute for options.

Reply:
Added value attribute!